packages/@uppy/url/src/Url.jsx

```diff
@@ -88,6 +91,8 @@ export default class Url extends UIPlugin {
       companionHeaders: this.opts.companionHeaders,
       companionCookiesRule: this.opts.companionCookiesRule,
     })
+
+    this.uppy.registerRequestClient(Url.requestClientId, this.client)
```

Comment:
Why only Url and not all other providers?

Reply:
Can we add a comment then why URL is the only exception?

Reply:
No, what i'm saying is that URL is not an exception. we do the same inside ProviderView and SearchProviderView

Reply:
Hmm, not sure what the comment would say.